.github/workflows/schedule.py

```diff
@@ -167,21 +166,32 @@ def create_issue(self, title_date: date) -> None:
         if issues:
             log.info('At least one matching issue already exists: %r', issues)
         else:
-            assignees = self.properties.get('assignees')
-            if assignees is not None:
-                flags.append(f'--assignee={assignees}')
             command = [
                 'gh', 'issue', 'create',
-                *flags,
+                *repo,
                 f'--title={title}',
-                f'--label={self.properties["labels"]}',
                 f'--body={self.body}'
             ]
+            edit_command = ['gh', 'issue', 'edit', *repo]
+            assignees = self.properties.get('assignees', [])
+            if assignees:
+                assignees = [f'--add-assignee={assignees}']
+            labels = self.properties.get('labels', [])
+            if labels:
+                labels = [f'--add-label={labels}']
             if self.dry_run:
                 log.info('Would run %r', command)
+                for option in [*assignees, *labels]:
+                    option = edit_command + ['ISSUE_NUMBER', option]
+                    log.info('Followed by %r', option)
             else:
                 log.info('Running %r', command)
-                subprocess.run(command, check=True)
+                issue_number = subprocess.run(command, check=True, capture_output=True)
+                issue_number = issue_number.stdout.decode().strip().rsplit('/', 1)[1]
+                for option in [*assignees, *labels]:
+                    option = edit_command + [issue_number, option]
```

Comment:
With this reassignment, `option` is a misleading variable name. We generally only reuse a variable via reassignment if the new value is semantically equivalent to the old, potentially after a transformation. This reassignment is not a transformation of one form into another. It composes something new from various parts.